### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jguoamz**
*Issue #, if available:*

*Description of changes:*

Updating readme to include Amazon documentation. See preview here: https://github.com/jguoamz/codeguru-reviewer/tree/patch-1

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Amazon CodeGuru Reviewer action can be triggered by a pull request, push, or sch
 * Push - Code quality and security recommendations are shown in the security tab
 * Manual / Scheduled runs - Code quality and security recommendations are shown in the security tab
 
-To add CodeGuru Reviewer into your CI/CD pipeline, do the following.
+You can read more about setup in [Amazon CodeGuru Reviewer document](https://docs.aws.amazon.com/codeguru/latest/reviewer-ug/working-with-cicd.html). To add CodeGuru Reviewer into your CI/CD pipeline, do the following.
```
Comment (amylmill) [inline README.md]: I do not have permission to edit or approve this PR. I suggest the following changes to the content.

For more information about setup, see [Create code reviews with GitHubActions](https://docs.aws.amazon.com/codeguru/latest/reviewer-ug/working-with-cicd.html) in the *Amazon CodeGuru Reviewer User Guide*. 

To add CodeGuru Reviewer to your CI/CD pipeline, follow the steps under the *Usage* section.

Comment (jguoamz) [reply README.md]: Fixed in latest commit

**Review by amylmill — CHANGES_REQUESTED** [review]
I introduced an error in my previous comment. Please add a space between "GitHub" and "Actions".

File: README.md [hunk 2]
```diff
@@ -11,7 +11,9 @@ Amazon CodeGuru Reviewer action can be triggered by a pull request, push, or sch
 * Push - Code quality and security recommendations are shown in the security tab
 * Manual / Scheduled runs - Code quality and security recommendations are shown in the security tab
 
-You can read more about setup in [Amazon CodeGuru Reviewer document](https://docs.aws.amazon.com/codeguru/latest/reviewer-ug/working-with-cicd.html). To add CodeGuru Reviewer into your CI/CD pipeline, do the following.
+For more information about setup, see [Create code reviews with GitHubActions](https://docs.aws.amazon.com/codeguru/latest/reviewer-ug/working-with-cicd.html) in the *Amazon CodeGuru Reviewer User Guide*.
```
Comment (yueny2020) [inline README.md:14]: please add space between `GitHub` and `Actions`.

**Comment by jguoamz** [comment]
> I introduced an error in my previous comment. Please add a space between "GitHub" and "Actions".

Fixed

File: README.md (hunk 2 above)
Comment (jguoamz) [reply README.md]: Fixed

**Review by amylmill — APPROVED** [review]

**Pull request merged**